### ISSUE TITLE
App modules: import core or alias app modules for Mail and modern keyboard

### ISSUE THREAD

**Issue by josephsl** [josephsl/wintenApps#70]
Hi,

The following depends on when NVAccess/nvda#13366 is approved and merged:

As part of NVDA Core pull request 13366, ap module loading process was refactored with two changes: letting add-ons register an app module name for modules that can be problematic when importing it (such as using a character that isn't supported by Python's import machinery), and removing NVDA Core app modules which as aliases to other app modules (such as "hxoutlook" alias for "hxmail"). Of these, the second change affects this add-on as the add-on relies on existence of alias app modules when importing app module contents (since the alias app module names are used in recent Windows 10 releases, and in extension, Windows 11). Therefore, detect both the core and the alias ap modules and import the core app module if loading the alias app module fails. This change will be part of copper 1 (March 2022 at the earliest) assuming that the mentioned pull request is included in NVDA 2022.1.

The following app modules are affected:

* Mail: hxoutlook
* Modern keyboard: textinputhost

Thanks.

**Comment by josephsl** [comment]
Hi,

Change of plan: go all the way in and import core app modules just to play safe and to ease NVDA pull request creation (pull requests are done using the core ap modules anyway).

Thanks.

**Comment by josephsl** [comment]
Finished with app module renames.
